packages/canary-bot/src/canary-bot.ts

```diff
@@ -117,4 +127,12 @@ export = (app: Probot) => {
       );
     }
   });
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  app.on('pubsub.message' as any, async (context: PubSubContext) => {
```

Comment:
I've found you can cast to `as '*'` and get the same effect without the `any`.

Reply:
Unfortunately we can not. flakybot and owl-bot both use `any` for the pubsub handler.